### DISCUSSION TITLE
What is the state of Android 12 support. Phantom process killer

### DISCUSSION THREAD

**Discussion by xplshn** [termux/termux-app#3387] [Q&A] [answered]
The issue was marked as closed and solved, but no thorough answer was made. Anyone knows if Termux will work nomally without modifying the Phantom killer in Android? https://github.com/termux/termux-app/issues/2366

**Comment by agnostic-apollo** [comment, marked as answer]
Read the first line of the issue.

Termux will not run fine if apps are running more than 32 processes, no other way than disabling killer.

**Reply by xplshn** [reply]
I read that Google had merged a flag to disable it. So I thought I wouldn't have to use ADB, okay then. Thanks

**Reply by agnostic-apollo** [reply]
Toggle is only available in Android 14 which is still in beta and the toggle (different from feature flag) also disables phantom killer even if it doesn't require adb/root.

**Reply by xplshn** [reply]
I figured it out, now I have the toggle on Dev settings, though its weird because I'm in A13. Maybe it got backported by LOS

**Reply by agnostic-apollo** [reply]
Likely using userdebug LOS build, which would show feature flags.

Android 14 toggle is different as already mentioned.

https://issuetracker.google.com/issues/205156966#comment120
